### PR TITLE
Add bazel BCR configuration files

### PR DESCRIPTION
**Pull request by jpsim**
Should allow https://github.com/apps/publish-to-bcr to push releases to BCR automatically.

### CODE REVIEW

File: .bcr/source.template.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "url": "https://github.com/jpsim/Yams/archive/refs/tags/{TAG}.tar.gz",
+    "integrity": "",
+    "strip_prefix": ""
```
Comment (keith) [inline .bcr/source.template.json:4]:
```suggestion
    "strip_prefix": "Yams-{TAG}"
```

File: .bcr/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+tasks:
+  verify_targets:
+    name: Verify targets
+    platform: macos
+    build_targets:
+      - '@yams//:Yams'
+      - '@yams//Tests/...'
```
Comment (keith) [inline .bcr/presubmit.yml:7]: can remove this since testing will build everything as long as there aren't build only targets in there, nbd either way

**Review by keith — APPROVED** [review]
looks good but you might want to test on linux too? https://github.com/keith/rules_multirun/blob/main/.bcr/presubmit.yml

**Comment by jpsim** [comment]
> looks good but you might want to test on linux too?

Is Swift installed on BCR's Linux CI?

**Comment by keith** [comment]
ah looks like you need this if so https://github.com/bazelbuild/bazel-central-registry/blob/2c81a61e94b31f3d5a8549d30a45f6de451ebf83/modules/rules_swift/1.5.1/presubmit.yml#L1-L4

**Review by keith — APPROVED** [review]

**Pull request merged**